dev/devicelab/lib/framework/cocoon.dart

```diff
@@ -64,26 +61,20 @@ class Cocoon {
     return _commitSha = result.stdout as String;
   }
 
-  /// Parse the local repo for the current running branch.
-  String _readCommitBranch() {
-    final ProcessResult result = processRunSync('git', <String>['rev-parse', '--abbrev-ref', 'HEAD']);
-    if (result.exitCode != 0) {
-      throw CocoonException(result.stderr as String);
-    }
-
-    return _commitBranch = result.stdout as String;
-  }
-
   /// Send [TaskResult] to Cocoon.
-  Future<void> sendTaskResult({String builderName, TaskResult result}) async {
+  Future<void> sendTaskResult({@required String builderName, @required TaskResult result, @required String gitBranch}) async {
```

Comment:
switch to positional arguments?

Reply:
Stylistically, I prefer named args when there are more than 1 arg. However, I don't see any guidance from effective dart on this. Thoughts?

Reply:
not sure about official guidance, i'll just add in the flutter tool, we prefer named args when there are 3 or more